stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/content/DefaultMessageComposerCenterContent.kt

```diff
@@ -170,12 +190,12 @@ internal class MessageComposerAttachmentsAdapter(
     /**
      * List of [MessageComposerAttachmentPreviewFactory] instances responsible for providing specific [View] for given [Attachment].
      */
-    internal var viewFactories: List<MessageComposerAttachmentPreviewFactory> = listOf()
+    var viewFactories: List<MessageComposerAttachmentPreviewFactory> = listOf()
 
     /**
      * Updates the list of currently displayed attachments. Re-renders all the attachments.
      */
-    internal fun setAttachments(attachments: List<Attachment>) {
+    fun setAttachments(attachments: List<Attachment>) {
```

Comment:
don't we need to say private due to explicit mode?

Reply:
Doesn't compile with private